lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -704,6 +704,23 @@ tokenize(String, Line, Column, OriginalScope, Tokens) ->
 previous_was_dot([{'.', _} | _]) -> true;
 previous_was_dot(_) -> false.
 
+unexpected_identifier_token([T | Rest] = String, Line, Column, Scope, Tokens) ->
+  AsNFKC = unicode:characters_to_nfkc_list(unicode:characters_to_nfc_list([T])),
```

Comment:
Do we need both characters_to_nfc_list and characters_to_nfkc_list?

Reply:
Ah -- I'd assume we don't 🤦